src/components/PostCard/PostCard.js

```diff
@@ -64,12 +66,12 @@ export default class PostCard extends React.Component {
     const hasImage = attachmentType === 'image' || false
 
     return (
-      <div ref='postCard'
-        onClick={!isEvent ? this.onClick : null}
```

Comment:
how come you are removing all these onclicks? i feel like they were there for a reason...

Reply:
I needed to rejuggle them all to ensure the right component captured the click. Othewise clicks would just open the postDetail, instead of allowing for any reactions to happen. Event's already had some modifications for this.